### PR TITLE
USA Phone number

### PR DESCRIPTION
**Pull request by kylelol**
Added a USAPhoneNumber case to the Trial struct.

### CODE REVIEW

File: SwiftCop/Trial.swift [hunk 1]
```diff
@@ -61,8 +63,9 @@ public enum Trial: TrialProtocol {
 		case .Email:
 			return { (evidence: String) -> Bool in
 				let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,4}"
-				let emailTest = NSPredicate(format:"SELF MATCHES %@", emailRegEx)
-				return emailTest.evaluateWithObject(evidence)
+				/*let emailTest = NSPredicate(format:"SELF MATCHES %@", emailRegEx)
```
Comment (andresinaka) [inline SwiftCop/Trial.swift:66]: please can you remove the comments?

File: SwiftCop/Trial.swift [hunk 2]
```diff
@@ -43,8 +44,9 @@ public enum Trial: TrialProtocol {
 			
 		case let .Format(regex):
 			return { (evidence: String) -> Bool in
-				let regexTest = NSPredicate(format:"SELF MATCHES %@", regex)
-				return regexTest.evaluateWithObject(evidence)
+				/*let regexTest = NSPredicate(format:"SELF MATCHES %@", regex)
```
Comment (andresinaka) [inline SwiftCop/Trial.swift:47]: please can you remove the comments?

**Comment by andresinaka** [comment]
Hey, thanks a lot for contributing!! Just one picky comment, can you change the spaces used to indent by tabs?

**Comment by kylelol** [comment]
sure what do you want the spacing to be set to?

**Comment by andresinaka** [comment]
the default tab spacing, just to keep the indent spaces consistent with the rest of the project... these are my xcode settings: 

screen shot 2015-12-16 at 9 09 26 am

**Pull request closed without merging**